### DISCUSSION TITLE
Training data seems old compare to what is available.

### DISCUSSION THREAD

**Discussion by Satch06** [Genuitec/Copilot4Eclipse-community#27] [Help] [answered]
Hello,
I have asked the same question via Copilot4Eclipse and VS Code using the same account to connect to Github Copilot. The answer provided by Copilot4Eclipse is using older training data (September 2021) than the VS Code one (Octobre 2023). Do you know any reason why Copilot4Eclipse is not using the last trained data ?

**Comment by wayneparrott** [comment, marked as answer]
Context has a lot to do with the quality of code generation from GitHub Copilot services. Can you share any details for how you qualified responses to specific datasets? 

Note that we do not have control over the LLM used by GitHub Copilot (yet). We have seen GitHub beta-ing access to the new 4-oh model which should be very helpful if not a game changer for supporting much more sophisticated solutions. 

Wayne
Copilot4Eclipse Team

**Reply by Satch06** [reply]
Hi,
The context was very simple. I just asked what is the last angular version. And the answer was:
![image](https://github.com/user-attachments/assets/b0ffe3cc-53fe-4178-9b9e-9c0f57e376ba)

whereas Copilot answer is :

![image](https://github.com/user-attachments/assets/3c69bcde-5d92-4de1-89b2-901845e942cc)
